cmd/generate-inventory/tmpl/model.gotmpl

```diff
@@ -14,25 +14,28 @@ import (
 	"github.com/FuturFusion/operations-center/internal/domain"
 )
 
+//
+//generate-expr: {{ .Name | pascalcase }}
+
 type {{ .Name | pascalcase }} struct {
-	ID          int
-	UUID        uuid.UUID
-	Cluster     string
+	ID          int `json:"id"`
+	UUID        uuid.UUID `json:"uuid"`
+	Cluster     string `json:"cluster"`
 {{- if .ServerIDByLocation }}
-	Server      string
+	Server      string `json:"server"`
 {{- end }}
 {{- if .HasProject }}
-	ProjectName string
+	ProjectName string `json:"project"`
 {{- end }}
 {{- if .HasParent }}
-	{{ .ParentName | pascalcase }}Name string
+	{{ .ParentName | pascalcase }}Name string `json:"{{ .ParentName | snakecase }}_name"`
```

Comment:
`ParentName` is coming from the config file and there the values are already snakecase by convention.


```suggestion
	{{ .ParentName | pascalcase }}Name string `json:"{{ .ParentName }}_name"`
```